packages/docs/docs/communications/async-encounters.md

```diff
@@ -0,0 +1,53 @@
+# Representing Asynchronous Encounters
+
+## Intro
+
+In healthcare, an "encounter" refers to any diagnostic or treatment interaction between a patient and provider. In traditional healthcare settings, this typically refers to an in-person visit, and is represented by the [Encounter](/docs/api/fhir/resources/encounter) resource.
+
+But in digital health, this can take on a variety of asynchronous forms, including SMS chains, in-app chat threads, or even an email exchange.
+
+In this guide, we'll show you how to represent these kinds of asynchronous encounters in FHIR.
+
+## Defining Sessions
+
+To get started, you'll first need to determine what determines a "session" in your care setting.
+
+A session could be a single SMS chain, or a single email thread. Many digital health apps ask the patient to explicitly initiate a messaging session with a provider.
+
+Alternatively, if your care setting has more of an rolling interaction model (e.g. a continuous text thread), you may choose to group all communications from the same day into a session.
+
+## Representing Sessions in FHIR
+
+Each session should be represented by an [Encounter](/docs/api/fhir/resources/encounter) resource. All of the messages that are part of this session should be represented as [Communication](#) resources and can be linked to the session using the `Communication.encounter` element.
+
```

Comment:
Consider an awesomic diagram here, or perhaps start with one from slides.  As this is a key point!

Reply:
Yeah good point